test/is/date-time.test.js

```diff
@@ -0,0 +1,46 @@
+import dateTime from "../../src/is/date-time";
+
+const { isPast, isFuture } = dateTime;
+
+describe("DateTime", () => {
+    it("should be an object with keys `isPast` and `isFuture` that are functions", () => {
```

Comment:
Overtesting.
Implicitly tested by the import and usage of method from `date-time.js`